client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,44 @@
+import Box from '@material-ui/core/Box';
+import useStyles from './useStyles';
+import { AppBar, Button, Paper } from '@material-ui/core';
+import LogoHeader from '../LogoHeader/LogoHeader';
+import logo from '../../Images/68f55f7799df6c8078a874cfe0a61a5e6e9e1687.png';
+import AuthMenu from '../AuthMenu/AuthMenu';
+import { User } from '../../interface/User';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const NavBar = ({ loggedInUser }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box style={{ height: 70 }}>
+      {/*Add Box parent to push other content down, can remove box to fix nav bar to top of screen*/}
+      <AppBar position="sticky" className={classes.appBar}>
+        <LogoHeader logo={logo} />
+        <Box className={classes.appBarButtonsWrapper}>
+          <Link to="/">
+            <Button className={classes.appBarButtons}>Notifications</Button>
+          </Link>
+          <Link to="/">
+            <Button className={classes.appBarButtons}>Messages</Button>
+          </Link>
+          <Link to="/">
+            <Button className={classes.appBarButtons}>My Jobs</Button>
+          </Link>
+          <Link to="/">
+            <Button className={classes.appBarButtons}>Requests</Button>
+          </Link>
```

Comment:
A few concerns here:
- Generally, we want to use material ui's `Link/Button` components to render `react-router` links. This gives us more flexibility and enables us to leverage all the tools material ui provides in terms of styling/consistency. We can achieve this by using the `component` prop 👍🏼
- This goes hand-in-hand with what we mentioned earlier when talking about `Box`es: Most material ui components come with props that impact their styling (i.e. `Button` has `color`, `variant` and `size` props; and much [more](https://v4.mui.com/api/button/#button-api)). Using those instead of pure CSS has many benefits (one of the more important being staying consistent with the [theme of the app](https://v4.mui.com/customization/theming/#theming)) and we should try to stick to them as much as possible.
- We could also modify this a bit to follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) just to adhere to best practices, it's not that big of a deal for this specific case.

Reply:
Thanks for the review. In this case are you saying it's better to do it this way:
 `import { Link as RouterLink } from 'react-router-dom';`
`import { Link as MUILink } from '@material-ui/core';`
`<MUILink component={RouterLink} to={'/'}>`

Also, for the DRY part, would it be better if I put the options into an array then mapped it to the buttons? 
Thanks I will make all the other changes.

Reply:
> <MUILink component={RouterLink} to={'/'}>
That's it, yes! You don't have to use a `Link` though, `Button`s have the `component` prop too!

> Also, for the DRY part, would it be better if I put the options into an array then mapped it to the buttons?
Yeah, that's exactly what I had in mind. Since there aren't many routes, it's okay to do either way... It might also be a good idea to abstract this away into a reusable component.